rocky/katalogus/views/boefje_setup.py

```diff
@@ -44,6 +47,7 @@ def form_valid(self, form):
         plugin = create_boefje_with_form_data(form_data, self.plugin_id, self.created)
 
         try:
+            logger.info("Creating boefje", event_code=800025, boefje=plugin)
```

Comment:
Perhaps only log the plugin ID or name instead of the complete structure? Like you do at other places

Reply:
I'm in favor of  sending the whole boefje to audit logs.

Reply:
This goes against the structured logging we try to do. You get a log message like:

```[info] Creating boefje                boefje=Boefje(id='5f569b52-7140-4c31-82e9-c3c09d73fad9', name='test5', version=None, authors=None, create
d='2024-10-03 14:15:47.319434', description='test', related=[], enabled=False, type='boefje', scan_level=SCAN_LEVEL.L0, consumes=set(), produces=set(), options=None, runnable_hash=None, interval=None, boefje_schema=None, oci_image='test', oci_arguments=[]) event_code=800025 ip=172.30.0.1 organization_code=test request_id=dcfb0bbc-5858-425c-9ceb-3be8121bd430 user_id=1```

Other than that it is ugly and logs a lot of uninteresting data, how are you supposed to easily filter all logs messages for a boefje with the name 'test5'? The whole idea of structured logging is that you can have filter with for example a key boefje_name and value 'test5'...